delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphView.scala

```diff
@@ -112,8 +112,8 @@ object BlazegraphView {
       description: Option[String],
       project: ProjectRef,
       uuid: UUID,
-      resourceSchemas: Set[Iri],
-      resourceTypes: Set[Iri],
+      resourceSchemas: ValidViewTypes,
```

Comment:
For `resourceSchemas`, the `Iri` defines the identifier of the schema 
For `resourceTypes`, the `Iri` defines the types of the resources eligible to index.
So we need a better name